packages/framework/src/Model/Product/Pricing/ProductCalculatedPriceRepository.php

```diff
@@ -70,14 +70,16 @@ public function saveCalculatedPrice(Product $product, PricingGroup $pricingGroup
      */
     public function createProductCalculatedPricesForPricingGroup(PricingGroup $pricingGroup)
     {
-        $query = $this->em->createNativeQuery(
+        $this->em->getConnection()->executeStatement(
             'INSERT INTO product_calculated_prices (product_id, pricing_group_id, price_with_vat)
             SELECT id, :pricingGroupId, :priceWithVat FROM products',
-            new ResultSetMapping()
+            [
+                'pricingGroupId' => $pricingGroup->getId(),
+                'priceWithVat' => null,
```

Comment:
Could not be the `NULL` set in query directly?

Reply:
Good point. fixed in 706c46209